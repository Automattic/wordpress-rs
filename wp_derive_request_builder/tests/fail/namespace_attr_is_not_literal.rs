#[derive(wp_derive_request_builder::WpDerivedRequest)]
#[Namespace(wp)]
#[SparseField(crate::SparseUserField)]
enum UsersRequest {
    #[contextual_get(url = "/users", params = &UserListParams, output = Vec<SparseUser>)]
    List,
}

fn main() {}
