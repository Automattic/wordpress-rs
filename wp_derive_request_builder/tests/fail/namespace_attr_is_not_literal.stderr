error: Expecting #[Namespace("_path_")] - Try wrapping the path in ""
 --> tests/fail/namespace_attr_is_not_literal.rs:2:13
  |
2 | #[Namespace(wp)]
  |             ^^
